cvcl/urls.py

```diff
@@ -30,5 +32,5 @@
     url(r'^envdefs/(?P<pk>\d+)/createvmdef$', VmDefinitionCreate.as_view(), name='envdefs.createvmdef'),  # instructor
     url(r'^vmdefinitions/(?P<pk>\d+)/update$', VmDefinitionUpdate.as_view(), name='vmdef.update'),  # instructor
     url(r'^vmdefinitions/(?P<pk>\d+)/delete$', VmDefinitionDelete.as_view(), name='vmdef.delete'),  # instructor
-    url(r'^courses/add_students/', DataView.as_view(), name='csv_upload'),
+    url(r'^courses/(?P<pk>\d+)/deleteStudent/(?P<user_pk>\d+)', RemoveStudent.as_view(), name='delete_user'),
```

Comment:
So you use `deleteStudent`, `RemoveStudent` and `delete_user` as 3 different names for a single thing in one line?